# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/tps/tps/testrunner.py
# Commit: 021c18645209
# Full Hash: 021c186452092d42720c7ba05f51299ad1026dd8
# Author: Sammy Khamis <skhamis@mozilla.com>
# Date: 2023-03-07 16:07:33
# Regressor Bug: 1818349
# File Overlap Count: 1
# Overlapping Files: testing/tps/tps/testrunner.py
# Description:
#   Bug 1818349 Part 2: Update tabs TPS tests to allow using data: schema r=markh
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171182
# ==============================================================================

diff -r 3326192e8db6 -r 021c18645209 testing/tps/tps/testrunner.py
--- a/testing/tps/tps/testrunner.py	Tue Mar 07 07:42:14 2023 +0000
+++ b/testing/tps/tps/testrunner.py	Tue Mar 07 07:42:14 2023 +0000
@@ -79,6 +79,8 @@
         # hrm - not sure what the release/beta channels will do?
         "xpinstall.signatures.required": False,
         "services.sync.testing.tps": True,
+        # removed data: schema restriction for easier testing of tabs
+        "services.sync.engine.tabs.filteredSchemes": "about|resource|chrome|file|blob|moz-extension",
         "engine.bookmarks.repair.enabled": False,
         "extensions.experiments.enabled": True,
         "webextensions.storage.sync.kinto": False,
